### ISSUE TITLE
What does predict_scores represent?

### ISSUE THREAD

**Issue by zhengkang128** [intel-isl/Open3D-ML#202]
Hi, I am training a KPFCNN model to run semantic segmentation inference and output the confidence scores for each points.
However, I notice that my output values, 'predict_scores', consists of values higher than 1, as well as negative scores, which should not be the case if 'predict_scores' refers to the confidence level (0-1) for each class.
May I also clarify what 'predict_scores' refer to?  

`       results_r = pipeline_r.run_inference(data)`
`print(results_r['predict_scores]`

(
![Screenshot from 2020-12-29 14-18-22](https://user-images.githubusercontent.com/72126038/103262709-c154e380-49e0-11eb-9ca4-3067e20f430d.png)

(My model consists of 2 distinct classes)

**Comment by germanros1987** [comment]
@YilingQiao could you please help with this?

**Comment by YilingQiao** [comment]
Hi, could you please tell us what framework (tf/torch) you are using? Did you change `test_smooth` or modify the network strucutre? Normally, the `predict_scores` should be between 0-1, because it's the moving exponential average of the output of `softmax()`.

**Comment by daanbl** [comment]
Hi, same issue here. I ran `pipeline.run_inference` on a pre-trained RandLA-Net (torch) on the validation set of Paris-Lille-3D (default config, code snippet from readme) and got this.
![Inference results](https://user-images.githubusercontent.com/12812413/112001588-66cead80-8b1f-11eb-949c-424d7dab42d7.png)

**Comment by YilingQiao** [comment]
> Hi, same issue here. I ran `pipeline.run_inference` on a pre-trained RandLA-Net (torch) on the validation set of Paris-Lille-3D (default config, code snippet from readme) and got this.
> ![Inference results](https://user-images.githubusercontent.com/12812413/112001588-66cead80-8b1f-11eb-949c-424d7dab42d7.png)

Thanks for your information. I just created PR #244 for it. You can just change `probs = result.cpu().data.numpy()` to `probs = probs.cpu().data.numpy()` in `randlanet.py`. A similar fix also applies to `kpconv.py`.
